mws/mws.py

```diff
@@ -57,6 +57,15 @@ class MWSError(Exception):
     response = None
 
 
+class InputError(Exception):
```

Comment:
No hard big issue with adding a new Exception type, but we should be cautious about it. It might be better to simply raise `ValueError` in cases where this type is being used, or to overwrite it as a, for instance, `MWSValueError`. Just a thought.